packages/devtools_app/test/visible_screens_test.dart

```diff
@@ -0,0 +1,183 @@
+      if (flutter) {
+        fakeServiceManager.availableLibraries
+            .add('package:flutter/src/widgets/binding.dart');
+      }
+      mockIsDebugFlutterApp(
+          fakeServiceManager.connectedApp, flutter && !profileMode);
+    }
+
+    testWidgets('are correct for Dart CLI app', (WidgetTester tester) async {
+      setupMockValues();
+
+      expect(
+          visibleScreenTypes,
+          equals([
+            // InspectorScreen,
+            TimelineScreen,
+            MemoryScreen,
+            PerformanceScreen,
+            DebuggerScreen,
+            NetworkScreen,
+            LoggingScreen,
+            if (codeSizeScreenEnabled) CodeSizeScreen,
+          ]));
+    });
+
+    testWidgets('are correct for Dart Web app', (WidgetTester tester) async {
+      setupMockValues(web: true);
+
+      expect(
+          visibleScreenTypes,
+          equals([
+            // InspectorScreen,
+            // TimelineScreen,
+            // MemoryScreen,
+            // PerformanceScreen,
+            DebuggerScreen,
+            NetworkScreen,
+            LoggingScreen,
+            if (codeSizeScreenEnabled) CodeSizeScreen,
+          ]));
+    });
+
+    testWidgets('are correct for Flutter (non-web) debug app',
+        (WidgetTester tester) async {
+      setupMockValues(flutter: true);
+
+      expect(
+          visibleScreenTypes,
+          equals([
+            InspectorScreen,
+            TimelineScreen,
+            MemoryScreen,
+            PerformanceScreen,
+            DebuggerScreen,
+            NetworkScreen,
+            LoggingScreen,
+            if (codeSizeScreenEnabled) CodeSizeScreen,
+          ]));
+    });
+
+    testWidgets('are correct for Flutter (non-web) profile app',
+        (WidgetTester tester) async {
+      setupMockValues(flutter: true, profileMode: true);
+
+      expect(
+          visibleScreenTypes,
+          equals([
+            InspectorScreen,
+            TimelineScreen,
+            MemoryScreen,
+            PerformanceScreen,
+            DebuggerScreen,
+            NetworkScreen,
+            LoggingScreen,
+            if (codeSizeScreenEnabled) CodeSizeScreen,
+          ]));
+    });
+
+    testWidgets('are correct for Flutter web debug app',
+        (WidgetTester tester) async {
+      setupMockValues(flutter: true, web: true);
+
+      expect(
+          visibleScreenTypes,
+          equals([
+            InspectorScreen,
+            // TimelineScreen,
+            // MemoryScreen,
+            // PerformanceScreen,
+            DebuggerScreen,
+            NetworkScreen,
+            LoggingScreen,
+            if (codeSizeScreenEnabled) CodeSizeScreen,
+          ]));
+    });
+
+    testWidgets('are correct for Flutter web profile app',
+        (WidgetTester tester) async {
+      setupMockValues(flutter: true, web: true, profileMode: true);
+
+      expect(
+          visibleScreenTypes,
+          equals([
+            InspectorScreen,
```

Comment:
here too - inspector and debugger should be disabled for profile builds

Reply:
Done!